src/main/kotlin/fi/metatavu/timebank/api/impl/PersonsApi.kt

```diff
@@ -21,12 +22,14 @@ class PersonsApi: PersonsApi, AbstractApi() {
     @Inject
     lateinit var personsTranslator: PersonsTranslator
 
-    override suspend fun listPersonTotalTime(personId: Int, timespan: Timespan?): Response {
+    override suspend fun listPersonTotalTime(personId: Int, timespan: Timespan?, before: LocalDate?, after: LocalDate?): Response {
```

Comment:
The submodule within this PR doesn't reference to an updated version of the spec and therefore this method doesn't seem to override anything. That is also causing the Google Cloud Build to fail.